.gitignore

```diff
@@ -59,4 +59,4 @@ venv/
 
 # Python packaging
 *.egg-info
-build/
+/build/
```

Comment:
Not sure why this change was made? I build BSK with the MuJoCo option on, and now I'm seeing several MuJoCo related build files as uncommitted files.